/**:
  ros__parameters:
    sitting_pose: [5.0, -30.0, 5.0, -30.0, 5.0, -30.0, 5.0, -30.0, 0.0, 0.0, 45.0, 45.0, 45.0, 45.0]
    standing_pose: [0.5, -0.5, 0.5, -0.5, 0.5, -0.5, 0.5, -0.5, 0.5, -0.5, 0.5, -0.5, 0.5, -0.5]
    kneeling_pose: [0.25, -0.25, 0.25, -0.25, 0.25, -0.25, 0.25, -0.25, 0.25, -0.25, 0.25, -0.25, 0.25, -0.25]
    lying_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]